src/common/row_set.h

```diff
@@ -149,25 +149,35 @@ class PartitionBuilder {
       max_n_tasks_ = n_tasks;
     }
   }
+  void AllocateForTask(size_t id) {
```

Comment:
Can you document when should these 2 new functions used?

Reply:
done

Reply:
Another function is redundant, so it was removed